packages/core/src/utils.js

```diff
@@ -394,13 +393,14 @@ export function getDisplayLabel(schema, uiSchema, rootSchema) {
       isMultiSelect(schema, rootSchema) ||
       isFilesArray(schema, uiSchema, rootSchema);
   }
+
   if (schema.type === "object") {
     displayLabel = false;
   }
   if (schema.type === "boolean" && !uiSchema["ui:widget"]) {
     displayLabel = false;
   }
-  if (uiSchema["ui:field"]) {
+  if (uiSchema && uiSchema["ui:field"]) {
```

Comment:
Can you explain again why this change was needed? I think we might have discussed this earlier, but I don't remember what we discussed earlier. If possible, we should remove it, but if we need it, we should add a test that covers it.